src/action/start.rs

```diff
@@ -14,20 +15,25 @@ const RCON_PASSWORD_LENGTH: usize = 32;
 
 /// Start lazymc.
 pub fn invoke(matches: &ArgMatches) -> Result<(), ()> {
+    // Parse bind address
+    let bind_addr: SocketAddr = matches.get_one::<String>("bind").unwrap().parse().unwrap();
+
     // Load config
     #[allow(unused_mut)]
-    let mut config = config::load(matches);
+    let mut configs = config::load(matches);
 
-    // Prepare RCON if enabled
-    #[cfg(feature = "rcon")]
-    prepare_rcon(&mut config);
+    for config in configs.iter_mut() {
+        // Prepare RCON if enabled
+        #[cfg(feature = "rcon")]
+        prepare_rcon(config);
 
-    // Rewrite server server.properties file
-    rewrite_server_properties(&config);
+        // Rewrite server server.properties file
+        rewrite_server_properties(config);
+    }
 
     // Start server service
-    let config = Arc::new(config);
-    service::server::service(config)
+    let configs_arc = configs.into_iter().map(Arc::new).collect();
```

Comment:
I don't think `Arc<HashMap<_, Arc<Config>>>` is desirable, as this is a very complex type.

I used `Arc<Config>` to make it easily sharable across async contexts with message passing.

I'll have to think about something that would be better here. I'm not sure right now. Maybe we need to create a custom type to manage these configurations for us rather than working with a hashmap directly.

Reply:
Added a new type `Router` in `src/router.rs` to deal with this, what do you think?